webrtc/videoroom/assets/src/room_ui.ts

```diff
@@ -43,27 +43,42 @@ export function getRoomId(): string {
 }
 
 export function addVideoElement(
-  _: MediaStreamTrack,
   stream: MediaStream,
   mute: boolean = false
-) {
+): void {
   let video = document.getElementById(stream.id) as HTMLVideoElement;
 
   if (!video) {
     video = document.createElement("video");
-    video.id = stream.id;
     const grid = document.getElementById("videos-grid")!;
     grid.appendChild(video);
 
     grid.className = `grid-${Math.min(2, grid.childNodes.length)}`;
   }
+
+  video.id = stream.id;
   video.srcObject = stream;
   video.autoplay = true;
   video.playsInline = true;
   video.muted = mute;
 }
 
-export function removeVideoElement(_: MediaStreamTrack, stream: MediaStream) {
+export function addAudioElement(stream: MediaStream): void {
```

Comment:
Are you sure that we aren't playing audio twice? If we add stream to video element and it contains audio track it will just play it. If we create another audio element and put there the stream it will play audio again.

Reply:
Thought about it. I didn't notice any problems so far

Reply:
The audio may be in sync and if 2 of them are playing we are introducing some overhead for the browser.

Reply:
True that, I would recall here if we notice any problems with browser performance or while limiting audio tracks to those that are the loudest. It has been working pretty well so far (I mean stable).